hepcrawl/spiders/desy_spider.py

```diff
@@ -278,17 +280,20 @@ def _get_marcxml_records(response_body):
 
     def _hep_records_from_marcxml(self, marcxml_records):
         def _create_json_record(xml_record):
-            object_record = create_record(etree.XML(xml_record))
             app = Flask('hepcrawl')
             app.config.update(
                 self.settings.getdict('MARC_TO_HEP_SETTINGS', {})
             )
             with app.app_context():
-                dojson_record = hep.do(object_record)
+                try:
+                    hep_record = marcxml2record(xml_record)
+                except Exception as e:
+                    return {'xml_record': xml_record, 'error': e,
+                            'traceback': traceback.format_tb(sys.exc_info()[2])}
                 base_uri = self.settings['SCHEMA_BASE_URI']
-                dojson_record['$schema'] = base_uri + 'hep.json'
+                hep_record['$schema'] = base_uri + 'hep.json'
```

Comment:
this is not needed if validation is not done in hepcrawl any more. You may need to synchronize with @iulianav about who removes this.